plugins/out_s3/s3.c

```diff
@@ -33,6 +33,7 @@
 #include <fluent-bit/flb_base64.h>
 #include <stdlib.h>
 #include <sys/stat.h>
+#include <stdbool.h>
```

Comment:
In Fluent Bit, we use `int` for bool as is standard in C and we use `FLB_TRUE` and `FLB_FALSE`. To check truth you should always do `var == FLB_TRUE` or `var == FLB_FALSE`. This is the standard in this code base